usaspending_api/etl/management/commands/update_total_funding_amount_sql.py

```diff
@@ -4,6 +4,7 @@
 """
 
 import logging
+from datetime import datetime
```

Comment:
My personal preference is to use [perf_counter](https://docs.python.org/3/library/time.html#time.perf_counter)